devops/ansible/roles/girder/templates/daemon/girder.service.j2

```diff
@@ -10,7 +10,19 @@ Restart=always
 {% if girder_virtualenv is defined %}
 ExecStart={{ girder_virtualenv }}/bin/python -m girder
 {% else %}
-ExecStart=python -m girder
+ExecStart={{ ansible_python.executable }} -m girder
+{% endif %}
+
+{% if girder_hardened %}
+# Hardened systemd options
+ProtectSystem=full
+ProtectHome=read-only
+ReadWriteDirectories={{ girder_path }} -{{ ansible_user_dir }}/.girder {{ girder_virtualenv|default("") }}
+
+#PrivateTmp=yes
+#PrivateDevices=yes
```

Comment:
These are additional, useful restrictions that I couldn't get working at the moment.